source/contracts/reporting/ReportingWindow.sol

```diff
@@ -17,11 +17,13 @@ import 'factories/MarketFactory.sol';
 import 'factories/RegistrationTokenFactory.sol';
 import 'reporting/Reporting.sol';
 import 'libraries/math/SafeMathUint256.sol';
+import 'libraries/math/RunningAverage.sol';
```

Comment:
I wonder if we could use one of the [other import formats](https://solidity.readthedocs.io/en/develop/layout-of-source-files.html#syntax-and-semantics) for libraries like this so we don't have to do `RunningAverage.Data`?  I haven't really thought it through or experimented, just came to mind since I learned about alternative import formats and aliasing today.

Reply:
I'll play around and see if i can make it something more palatable